src/main/java/exception/AnonymousException.java

```diff
@@ -0,0 +1,27 @@
+package exception;
+
+/**
+ * Thrown to notify user that enter a
+ * non-existing command.
+ */
+public class AnonymousException extends DukeException {
+    private String input;
+
+    public AnonymousException(String input) {
+        this.input = input;
+    }
+
+    /**
+     * Returns a short description of this throwable.
+     * The result is the concatenation of the name of
+     * the incorrect commend and a suggestion to run
+     * the help command.
```

Comment:
Comments could be more concise. By convention usually the "how" is not really included in comments.